zmsadmin/scss/custom.scss

```
@@ -124,4 +124,163 @@
 
 .ticketprinter-config__advanced-checkbox {
     margin-right: 8px;
+}
+
+/* Built via `npm run css` (scss/admin.scss → this file). Not custom_scss/admin.scss. */
+.queue-table-free-text,
+.process-free-text {
+    word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)